environs/config/config.go

```diff
@@ -634,6 +642,23 @@ func Validate(cfg, old *Config) error {
 		}
 	}
 
+	if raw, ok := cfg.defined[ContainerInheritProperiesKey].(string); ok && raw != "" {
+		rawProperties := strings.Split(raw, ",")
+		propertySet := set.NewStrings()
+		for _, prop := range rawProperties {
+			propertySet.Add(strings.TrimSpace(prop))
+		}
+		if propertySet.Contains("users") {
```

Comment:
I think we need a whitelist of supported attributes. Plus a blacklist of disallowed cloud init passthrough attributes.

Reply:
With agreement, a whitelist will be used to allow: "apt-primary", "apt-sources", "apt-security", "ca-certs".  The first 3 values refer to specific second level keys within "apt"